tests/test_config.py

```diff
@@ -176,6 +181,28 @@ def test_raises_int_caster():
         int_caster.cast("test")
 
 
+def test_float_caster():
+    float_caster = FloatCaster()
+    assert float_caster.cast("3.14") == 3.14
+    assert float_caster.cast("3,14") == 3.14
```

Comment:
A thing to note: that `atof` function works only with the user's current locale, so if you have English locale selected that uses `.` as the decimal point, this won't pass.